### ISSUE TITLE
Recommendation for OCR feature

### ISSUE THREAD

**Issue by burakyuksek** [nvaccess/nvda#10656]
### Is your feature request related to a problem? Please describe.
No, it's not related to a problem.
### Additional context
I very often use OCR while playing mainstream games. One of the enhancements I can suggest is, you should make a setting, when turned on, that outputs the OCR result to the NVDA currsor rather than a virtual window, so we can use NVDA up/down left/right arrows to review the result while continuing to play our game.
to give an example, let's say I'm driving a car and want to check my speed. In the current way, I have to press NVDA+r, the OCR result comes into a virtual window, so I need to let the throttle off, and check information. While I am checking information I cannot pause the game, so the car slows down. But if the result is outputted to the NVDA currsor, I can press escape to pause the game, review information with NVDA up down/left right, then resume my game. Also the currsor can act like a buffer of sorts, so unless I switch to a different window or perform another OCR, I can always check the last OCR result and remember things more clearly.

**Comment by Brian1Gaff** [comment]
This sounds like add on territory to me, as it is very specific. Have you 
found a lot of games where OCR actually does anything really useful? Maybe 
it might be better if the game makers considered outputting their info as 
text in the first place. I know , probably a long shot, but...
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by burakyuksek** [comment]
Of course, OCR is very very useful for mainstream games, as it is needed for reading every peace of text the game outputs to the screen.

**Comment by DrSooom** [comment]
@Brian1Gaff: Don't forget that NVDA is also used by visually impaired people who are able to read text visually, but maybe have issues with it due the used font, a too low contrast or the amount of text (keyword: Visual Novels).

**Comment by leonardder** [comment]
This shouldn't be very difficult to accomplish I think. Currently, the focus is changed to the OCR object. It should be enough to set it just as the navigator object instead.

**Comment by Flash109** [comment]
Yeah, it would be very great, if this could be done, so that the information can be erfreshed/updated without building this virtual document again over and over.

**Comment by Adriani90** [comment]
cc: @vortex1024, @jcsteh

**Comment by Adriani90** [comment]
The OCR is refreshing now continuously without having to build the virtual window again and again. Can we consider this fixed now in NVDA 2023.3 Beta?
cc: @leonardder

**Comment by jcsteh** [comment]
The other part of this request that isn't yet satisfied is the ability to have the OCR result accessible via the review cursor without moving the (virtual) focus.
